src/bp/ui-studio/src/web/views/Nlu/intents/UtterancesEditor.tsx

```diff
@@ -179,8 +180,8 @@ export class UtterancesEditor extends React.Component<Props, State> {
     const node: Node = editor.value.getIn(['document', 'nodes', utterance, 'nodes', block])
     const selectedTxt = node.text.substring(from, to)
     // // We're trimming white spaces in the tagging (forward and backward)
-    from += selectedTxt.length - selectedTxt.trimStart().length
-    to -= selectedTxt.length - selectedTxt.trimEnd().length
+    from += selectedTxt.length - selectedTxt.trimLeft().length
```

Comment:
whatsup with this change? are those equivalent?

Reply:
Yes équivalent, both are supported by browsers, simply in studio seems like we have a different es target version and this switch made it pass